### PR TITLE
[animations] README updates

### PR DESCRIPTION
**Pull request by shihaohong**
See https://github.com/shihaohong/packages/tree/animation-readme-examples/packages/animations for what the README looks like on GitHub.

### CODE REVIEW

File: packages/animations/README.md [hunk 1]
```diff
@@ -10,6 +10,89 @@ Currently, the following animated effects are available in this library:
```
Comment (goderbauer) [inline packages/animations/README.md]: Can you remove the line above where we claim that this is under heavy development?

File: packages/animations/README.md [hunk 2]
```diff
@@ -10,6 +10,89 @@ Currently, the following animated effects are available in this library:
 
 ### Material's [Open Container Transitions](https://material.io/design/motion/choreography.html#transformation)
```
Comment (goderbauer) [inline packages/animations/README.md]: Having the link in the title actually looks a little strange. Maybe we should have a dedicated link to the spec in the text below?

File: packages/animations/README.md [hunk 3]
```diff
@@ -10,6 +10,89 @@ Currently, the following animated effects are available in this library:
...
 
-Tapping on a container  (e.g. a card or a button) will expand the container to reveal more information.
+Tapping on a container (e.g. a card or a button) will expand the container to reveal more information.
 
-*TODO(goderbauer): Add example videos of this effect.*
+1) Fade Transition
+
+- Card Example
+
+<p align="center">
```
Comment (goderbauer) [inline packages/animations/README.md]: Having just one demo per line looks a little "lost". Maybe we should have three or for demo gifs next to each other with a short subtitle?

File: packages/animations/README.md (hunk 2 above)
Comment (shihaohong) [reply packages/animations/README.md]: We can do that but not yet, since the actual links are not available yet. I'll remove them for now

**Pull request merged**